test_scripts/Defects/6_2/1384/003_HMI_IsReady_response_TTS_interface.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1384
+--
+-- Description: SDL doesn't check result code on TTS.IsReady response from HMI
+--
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- 2) SDL receives TTS.IsReady (error_result_code, available=true) from the HMI
+-- 3) App is registered and activated
+-- In case:
+-- 1) App requests Speak RPC
+-- SDl does:
+-- 1) respond with 'UNSUPPORTED_RESOURCE, success:false,' + 'info: TTS is not supported by system'
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Defects/6_2/1384/common')
+
+--[[ Local Variable ]]
```

Comment:
Here should be `Local Variables` in case you set block name according to number items in the block(`Variable` or `Variables`)
But in general, this blocks has static names `Local Variables` and `Local Functions` and not depends on items number.